stock_picking_invoice_link/views/account_invoice_view.xml

```diff
@@ -16,6 +16,7 @@
                     type="object"
                     attrs="{'invisible': [('picking_ids', '=', [])]}"
                     string="Pickings"
+                    groups="stock.group_stock_user"
```

Comment:
This should be on the view, not here, or the same access errors will arise

Reply:
Actually there is no access error, I think the smart-button should not be shown if the user does not have inventory permissions.

Reply:
But is this needed then?

Reply:
IMO yes, show a smart-button to see pickings if the user can't access, I think it can only confuse.

Reply:
Billing users have explicit access to `stock.picking` so I'd this isn't needed.